src/Dto/AssetDto.php

```diff
@@ -11,6 +11,8 @@
 final class AssetDto
 {
     private string $value;
+    // the Symfony Asset package name which this asset belongs to
+    private string $packageName;
```

Comment:
I think this should have initial (null) value or we will have runtime errors when accessing uninitialized value in case `setPackageName` is never called.
```suggestion
    private ?string $packageName = null;
```